.eslintignore

```diff
@@ -0,0 +1 @@
+/src/user/user.controller.ts
```

Comment:
I think we should revert this. This will prevent user.controller.ts from being checked by the linter. We don't want that.